docs/data/stashcache/install-cache.md

```diff
@@ -206,15 +206,20 @@ To use HTTPS:
 
 1.  Add a certificate according to the [instructions above](#ensure-the-xrootd-service-has-a-certificate)
 
-1.  Create a file named `/etc/xrootd/config.d/11-cache-https.cfg` with the following contents:
+1.  Uncomment `set EnableVoms = 1` in `/etc/xrootd/config.d/10-osg-xrdvoms.cfg`
 
-        # Support HTTPS access to unauthenticated cache
-        if named stash-cache
-          http.cadir /etc/grid-security/certificates
-          http.cert /etc/grid-security/xrd/xrdcert.pem
-          http.key /etc/grid-security/xrd/xrdkey.pem
-          http.secxtractor /usr/lib64/libXrdLcmaps.so
-        fi
+!!! note Upgrading from OSG 3.5
```

Comment:
```suggestion
!!! note "Upgrading from OSG 3.5"
```

I thought that admonition titles had to be enclosed in double quotes

Reply:
Thanks, I forgot about that.